pkgs/top-level/python-packages.nix

```
@@ -497,6 +497,8 @@ in {
 
   codespell = callPackage ../development/python-modules/codespell { };
 
+  crypto-commons = disabledIf (!isPy3k) (callPackage ../development/python-modules/crypto-commons { });
```

move this into the expression please:
```
  disabled = !isPy3k;
```
```suggestion
  crypto-commons = callPackage ../development/python-modules/crypto-commons { };
```